bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -25,9 +25,6 @@
   {% endif %}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
Yes I can switch to page_css and there is no issue besides the fact that it inherits template-resp.css as well. Doesn't change the layout, but wondering if we even need template-resp.css at all? This may be another PR. 

Should we update or leave as is? @jpetto. I'm tempted to update, even if it adds this template-resp.css.